versions.tf

```diff
@@ -8,6 +8,9 @@ terraform {
   # 4.7.0 to 4.9.1 has a security regression: new repositories created via a  
   # template have a public visibility. Has been fixed in 4.9.2.
   required_providers {
-    github = ">= 4.5, < 5.0, != 4.7.0, != 4.8.0, != 4.9.0, != 4.9.1"
+    github = {
+      source  = "integrations/github"
+      version = ">= 4.5, < 5.0, != 4.7.0, != 4.8.0, != 4.9.0, != 4.9.1"
+    }
```

Comment:
The required terraform version needs to be increased, too..

afair lowest version supporting this is `0.13`